### ISSUE TITLE
Distribution on conda?

### ISSUE THREAD

**Issue by liamhuber** [sntioudis/papreca#4]
Going through the checklist on https://github.com/openjournals/joss-reviews/issues/6714, I got to:

> *Installation instructions:* Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.

I'm happy to check the box, because installation instructions were clear and I didn't have any hiccups, but it got me thinking about the automated bit.

I'm a big fan of conda-forge, so what I'd ideally like to do is just `conda install -c conda-forge papreca` and be done. Going back to the installation docs, the only big dependency is `lammps`. There is [already a conda package for this](https://github.com/conda-forge/lammps-feedstock/tree/main/recipe), which is an excellent start. Digging into the build directions, the feedstock and your instructions are extremely similar, up until `-DBUILD_LIB=on DBUILD_SHARED_LIBS=off -DBUILD_STATIC_LIBS=on`, which conflicts with [`D BUILD_LIB=ON -D BUILD_SHARED_LIBS=ON`](https://github.com/conda-forge/lammps-feedstock/blob/15afb3766e4971ebce1769d7e58e7f3e821ca023/recipe/build.sh#L91).

Nonetheless, it seems like the lammps dependency is 90% of the way to where you want it. I haven't managed a compiled-language conda feedstock (only no-arch python stuff), so I'm not sure what sort of choices there are for flags, or whether one would simply need to make a new `lammps-static` feedstock or something. And you'd need to follow a similar pattern for building papreca itself, but you can see in the lammps example that most of the build script complexity comes from the large number of build flags lammps has and supporting different architectures, so for linux-only papreca this is probably not too bad.

Not urgently needed, but would be nice.

**Comment by sntioudis** [comment]
It looks like the _LAMMPS static library (liblammps.a)_ is not built if LAMMPS is installed through the [conda-forge route](https://docs.lammps.org/Install_conda.html). I just verified that by building LAMMPS on my workstation through conda-forge.

Papreca uses the _static LAMMPS library_, hence, I reckon that a new lammps-static feedstock would have to be made.

**Comment by jfaraudo** [comment]
I also agree with @liamhuber , it will be great to have this sofware available with conda. It makes life much easier to users. I am also thinking about students, in my undergrad or summer courses it is usually not a big deal to ask people to install something from conda but doing more than that could be complicated. It is not urgent, but it will be great one day to have this possibility.

**Comment by sntioudis** [comment]
PAPRECA now links to a shared LAMMPS library (liblammps.so) which facilitates distribution through conda (we can now use the already available precompiled conda packages for LAMMPS).
